zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -0,0 +1,84 @@
+from zigpy import types as t
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import (
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.lighting import Color
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.xiaomi import BasicCluster, OppleCluster, XiaomiCustomDevice
+
+
+class LightOppleCluster(OppleCluster):
```

Comment:
This can be changed to inherit from `XiaomiAqaraE1Cluster` then.
Also, maybe rename the cluster to `OppleClusterLight`?

Reply:
Done